cmake/FindPDAL.cmake

```diff
@@ -17,28 +17,26 @@
 # and then again with no specified paths to search the default
 # locations. When an earlier FIND_* succeeds, subsequent FIND_*s
 # searching for the same item do nothing.
+
 FIND_PATH(PDAL_INCLUDE_DIR pdal.hpp
-  "$ENV{LIB_DIR}/include/"
+  "$ENV{LIB_DIR}/include"
   "$ENV{LIB_DIR}/include/pdal"
+  "/usr/include/pdal"
   c:/msys/local/include
   NO_DEFAULT_PATH
   )
-FIND_PATH(PDAL_INCLUDE_DIR pdal.hpp)
```

Comment:
I would keep it, see comment on lines 14-19. (we have the same in other Find* files)

Reply:
Ok, reverted